packages/api/src/entities/User.ts

```diff
@@ -46,6 +47,9 @@ export class User extends Node<User> {
   @Property({ columnType: 'timestamp', nullable: true })
   contributionsLastCheckedAt?: Date;
 
+  @OneToMany({ entity: () => Contribution, mappedBy: 'author' })
+  contributionList? = new Collection<Contribution>(this);
```

Comment:
This shouldn't be optional and we shouldn't be assigning a value here
```suggestion
  contributionList: IdentifiedReference<Contribution>[];
```

After making the two changes above, add this to the constructor:
``` typescript
this.contributionList = [];
```